src/SourceLocator/StaticClassMapSourceLocator.php

```diff
@@ -49,10 +52,6 @@ protected function createLocatedSource(Identifier $identifier) : ?LocatedSource
             return null;
         }
 
-        $fileContents = file_get_contents($classFile);
-
-        Assert::that($fileContents)->string();
```

Comment:
`file_get_contents` may return `false`, so assertion should remain

Reply:
Similar to the above, now using `Safe\file_get_contents`